api/rs/slint/mcu.md

```diff
@@ -80,42 +76,42 @@ in a format that's suitable for the software based renderer we're going to use.
 
 ## Application Structure
 
-Typically, a graphical application in hosted environments is comprised of at least three different tasks:
+Typically, a graphical application in hosted environments has at least three different tasks:
 
- * Receives user input from operation system APIs.
- * Reacts to the input by performing application specific computations.
- * Renders an updated user interface and presents it on the screen using device-independent operating system APIs.
+ * Receive user input from operation system APIs.
+ * React to the input by performing application specific computations.
+ * Render an updated user interface and presents it on the screen using device-independent operating system APIs.
```

Comment:
I remember discussing this with Kavindra when we wrote it and I also started with "Receive", "React", and "Render" and she pointed out that the tense should be consistent by using "receives", "reacts", and "renders". It makes sense to me to preserve the present tense of "has". Why did you change it to infinite?